regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,6 +26,7 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
+        $('table').stickyTableHeaders();
```

Comment:
Right. I'm saying that definitions, internal citations, etc. have event listeners added on page load (both hard refresh and via AJAX). How do those work? Should this work the same way rather than adding another step to the init script?